test/integration/commands/apply/index.test.ts

```diff
@@ -106,6 +106,8 @@ describe('create command', () => {
     .stdout({ print: true })
     .runApplyCommand(() => testContextInvalidToken)
     .it('should fail applying if token is invalid', (ctx) => {
-      expect(ctx.stdout).to.contain('Error: An error occurred while adding an entry.')
+      expect(ctx.stdout).to.contain(
```

Comment:
We could delete this "integration" test since we do the exact same thing, and they are expensive tests